### PR TITLE
Add Origin-Agent-Cluster article

### LINKED ISSUE #3795: content: Origin isolation [2020-10-15]
* [thread](https://groups.google.com/a/google.com/g/webdev-content/c/-dDIqsHcw8E)
* [draft](https://docs.google.com/document/d/1HfZTTXFP8itW8-Z1HuuMap1EpNtdlfa0eAulsuTk6eM/edit?usp=sharing)

### PR DESCRIPTION
**Pull request by domenic**
Fixes #3795.

### CODE REVIEW

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Requesting performance isolation with the Origin-Agent-Cluster header
+subhead: A new HTTP response header to limit domain-wide scripting and request dedicated resources from the browser.
+authors:
+  - domenic
+description: >
+  The Origin-Agent-Cluster header cuts off synchronous access to other origins on the same domain, and hints to the browser to give your origin dedicated resources.
+date: 2021-01-11
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:9]:
```suggestion
date: 2021-01-30
```

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+hero: hero.jpg
+alt: A cluster of grapes
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - performance
+  - memory
+---
+
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:17]: Please break all lines at 100 characters. If you're using VS Code, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) makes it pretty easy to do. The [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110?hl=en&pann=docs_addon_widget) Google Docs Add-On also supports this.

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
+---
+layout: post
+title: Requesting performance isolation with the Origin-Agent-Cluster header
+subhead: A new HTTP response header to limit domain-wide scripting and request dedicated resources from the browser.
+authors:
+  - domenic
+description: >
+  The Origin-Agent-Cluster header cuts off synchronous access to other origins on the same domain, and hints to the browser to give your origin dedicated resources.
+date: 2021-01-11
+hero: hero.jpg
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:10]: Please optimize this with tinyjpg.com (to keep the size of the repository down)

P.S. I like the "cluster" reference

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+## Feedback
+
+We'd love to hear from you if you're using, or considering using, the `Origin-Agent-Cluster` header. Your public interest and support helps the Chrome team prioritize features and show other browser vendors how important they are. Tweet at [@ChromiumDev](https://twitter.com/chromiumdev) and let us know your thoughts and experiences.
+
+If you have more questions about the specification, or the details of how the feature works, you can file an issue on the [HTML Standard GitHub repository](https://github.com/whatwg/html). And if you encounter any issues with Chrome's implementation, you can file a bug at [new.crbug.com](https://bugs.chromium.org/p/chromium/issues/entry?components=Internals%3ESandbox%3ESiteIsolation) with the Components field set to `Internals>Sandbox>SiteIsolation`.
+
+## Learn more
+
+To learn more about origi-keyed agent clusters, you can dive into the details at these links:
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:143]:
```suggestion
To learn more about origin-keyed agent clusters, you can dive into the details at these links:
```

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+* The user, or code running on a different site, might have already navigated to a site-keyed page on your origin, causing the [consistency guarantee](#how-to-use) to kick in and the `Origin-Agent-Cluster` header to be ignored entirely.
+
+For these reasons, it's important not to think of origin-keyed agent clusters as a security feature. Instead, it's a way of helping the browser prioritize resource allocation, by hinting that your origin would benefit from dedicated resources (and that you're willing to give up [certain features](#limitations) in exchange).
+
+## Feedback
+
+We'd love to hear from you if you're using, or considering using, the `Origin-Agent-Cluster` header. Your public interest and support helps the Chrome team prioritize features and show other browser vendors how important they are. Tweet at [@ChromiumDev](https://twitter.com/chromiumdev) and let us know your thoughts and experiences.
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:137]:
```suggestion
Chrome love to hear from you if you're using, or considering using, the `Origin-Agent-Cluster` header. Your public interest and support helps the Chrome team prioritize features and show other browser vendors how important they are. Tweet at [@ChromiumDev](https://twitter.com/chromiumdev) and let Chrome DevRel know your thoughts and experiences.
```
Avoiding "we" and just making it explicit instead

**Review by kaycebasques — CHANGES_REQUESTED** [review]
Hi @domenic just a couple clerical requested changes

File: src/site/content/en/blog/origin-agent-cluster/index.md (hunk 3 above)
Comment (domenic) [reply src/site/content/en/blog/origin-agent-cluster/index.md:13]: Thanks, and done! I feel like the guidance on https://web.dev/handbook/markup-media/ used to say no optimization, but now it's updated...

File: src/site/content/en/blog/origin-agent-cluster/index.md (hunk 2 above)
Comment (domenic) [reply src/site/content/en/blog/origin-agent-cluster/index.md:20]: Done!

File: src/site/content/en/blog/origin-agent-cluster/index.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+title: Requesting performance isolation with the Origin-Agent-Cluster header
+subhead: >
+  A new HTTP response header to limit domain-wide scripting and request dedicated resources from the
+  browser.
+authors:
+  - domenic
+description: >
+  The Origin-Agent-Cluster header cuts off synchronous access to other origins on the same domain,
+  and hints to the browser to give your origin dedicated resources.
+date: 2021-01-30
```
Comment (kaycebasques) [inline src/site/content/en/blog/origin-agent-cluster/index.md:12]:
```suggestion
date: 2021-02-01
```

**Review by kaycebasques — APPROVED** [review]

**Pull request merged**